README.md

```diff
@@ -18,6 +18,7 @@ require 'craigslister'
 client = Craigslister.new(
   item: 'Couch',
   area: 'austin', # optional (defaults to 'sfbay')
+  section: 'motorcycles', # optional (defaults to 'sss')
   low:  500,      # optional
   high: 2000,     # optional
 )
```

Comment:
Hmm maybe we could just put a table in the readme? where they could figure it out via the url? I'm thinking might be a bit of maintenance if we had a lookup table if they happen to change the query params